resources/views/components/productlist.blade.php

```diff
@@ -24,35 +30,37 @@
 
                     <div slot="renderNoResults"></div>
 
-                    <div class="relative" slot="render" slot-scope="{ data, loading }" v-if="!loading && data?.length">
+                    <div class="mt-5" slot="render" slot-scope="{ data, loading }" v-if="!loading && data?.length">
                         <slider>
                             <div slot-scope="{ navigate, showLeft, showRight, currentSlide, slidesTotal }">
-                                <div class="-mx-2 flex mt-5 overflow-x-auto snap-x scrollbar-hide scroll-smooth snap-mandatory" ref="slider">
-                                    <template v-for="item in data">
-                                        @include('rapidez::listing.partials.item', ['slider' => true])
-                                    </template>
+                                <div class="relative">
+                                    <div ref="slider" class="*:sm:w-1/2 *:md:w-1/3 *:xl:w-1/3 *:px-0.5 *:sm:px-2 *:shrink-0 *:snap-start -mx-2 -mx-4 flex snap-x snap-mandatory overflow-x-auto scroll-smooth scrollbar-hide sm:-mx-1">
+                                        <template v-for="item in data">
+                                            @include('rapidez::listing.partials.item', ['slider' => true])
+                                        </template>
+                                    </div>
+                                    <x-rapidez::button.slider
+                                        class="absolute left-0 top-1/2 sm:-translate-x-1/2 -translate-y-1/2"
+                                        v-if="showLeft"
+                                        v-on:click="navigate(currentSlide - 1)"
+                                        :aria-label="__('Prev')"
+                                    >
+                                        <x-heroicon-o-chevron-left :defer="false" class="size-6 shrink-0"/>
```

Comment:
We defer this as it's cached? With multiple sliders on a page not really great. Can we fix this differently?